src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -11,14 +12,66 @@
 from alchemtest.amber import load_invalidfiles
 from alchemtest.amber import load_bace_example
 from alchemtest.amber import load_bace_improper
+from alchemtest.amber import load_single_u_nk_file
+from alchemtest.amber import load_single_dHdl_file
```

Comment:
Will not be needed: see https://github.com/alchemistry/alchemtest/pull/66#discussion_r969934092 and https://github.com/alchemistry/alchemtest/pull/66#discussion_r969936058 for how to access single files.

Reply:
Removed in the last commit!